src/components/admin/article/ArticleController.jsx

```diff
@@ -59,6 +59,40 @@ export default class ArticleController extends FalcorController {
     }, 500);
   }
 
+  save(jsonGraphEnvelope, processedAuthors, articleSlug, falcorData) {
+    // Update the values
+    this.safeSetState({ saving: true });
+    const updatePromises = [this.falcorUpdate(jsonGraphEnvelope)];
+
+    if (processedAuthors !== null) {
+      updatePromises.push(this.falcorCall(
+          ['articles', 'bySlug', articleSlug, 'authors', 'updateAuthors'],
+          [falcorData.id, processedAuthors],
+          [['name'], ['slug']]
+        )
+      );
+    }
+
+
```

Comment:
Do we this extra newline?

Reply:
I'm passing Linting, but I'm happy to remove if need be.